codecs/libavcodec.rs

```diff
@@ -335,13 +335,14 @@ impl AvFrame {
                                        true).unwrap()
                                             .linesize;
         unsafe {
-            slice::from_raw_mut_buf(&(*self.frame).data[channel], len as usize)
+            slice::from_raw_parts(*&(*self.frame).data[channel], len as usize)
         }
     }
 }
 
 pub struct AvPacket<'a> {
     packet: ffi::EitherAVPacket,
+    phantom: ::std::marker::PhantomData<&'a[u8]>,
```

Comment:
uber-nit: put a space before `[u8]`
Also I think I'd prefer just importing `PhantomData` at the top of the file.